roles/setup/tasks/setup-container.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Enable linger for {{ receptor_user }}
+  become: true
+  ansible.builtin.command:
+    cmd: "loginctl enable-linger {{ receptor_user }}"
+  changed_when: false
+
+- name: Get UID of {{ receptor_user }}
+  ansible.builtin.command:
+    cmd: "id -u {{ receptor_user }}"
+  register: receptor_uid
+  changed_when: false
+
+- name: Setup container
+  become: true
+  become_user: "{{ receptor_user }}"
+  environment:
+    XDG_RUNTIME_DIR: "/run/user/{{ receptor_uid.stdout }}"
+  vars:
+    receptor_uid_directory: "/run/user/{{ receptor_uid.stdout }}"
+  block:
+    - name: Enable podman socket
+      ansible.builtin.systemd:
+        name: podman.socket
+        state: started
+        enabled: true
+        scope: user
+
+    - name: Pull the latest release of receptor from Quay
```

Comment:
We should not mention the registry here. Moreover, this should allow to use credentials to pull the container from private repos

Reply:
Updated.

I added an ability to lookup a file if it exists locally with the `receptor_image_authfile`. If it is not set, it is ignored.
The registry bundle may need to be updated to pass in the authfile, and if it is, we then may need to also update this collection again to create the authfile in the instance if it is contained in the bundle.